front/cypress/e2e/project_description_builder/project_description_builder_preview_toggle.cy.ts

```diff
@@ -108,16 +102,13 @@ describe('Project description builder preview', () => {
     cy.visit(
       `/admin/project-description-builder/projects/${projectId}/description`
     );
-    cy.get('div#ROOT');
-    cy.wait(1000);
-
-    // Open text widget
-    cy.get('.e2e-text-box').should('exist');
-    cy.get('.e2e-text-box').click();
-
-    // Edit text again
-    cy.get('#quill-editor').click();
-    cy.get('#quill-editor').type('Sample text.', { force: true });
+    cy.get('#e2e-draggable-text').dragAndDrop('#e2e-content-builder-frame', {
+      position: 'inside',
+    });
+    cy.wait(5000);
```

Comment:
This is a pretty long wait - do we need it to be this long? :)

Reply:
You are right, I don't think it should be this long. I shortened it.